.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this Pull Request and if possible provide links showcasing the changes.]
```

Comment:
Any particular reason for upper case **pull request**?

Reply:
It was to emphasize on PR, but I'll change it to lower case. That seems better.